public/app/features/inspector/utils/download.ts

```diff
@@ -30,7 +30,7 @@ export function downloadLogsModelAsTxt(logsModel: Pick<LogsModel, 'meta' | 'rows
   textToDownload = textToDownload + '\n\n';
 
   logsModel.rows.forEach((row) => {
-    const newRow = dateTimeFormat(row.timeEpochMs, { defaultWithMS: true }) + '\t' + row.entry + '\n';
```

Comment:
You probably need to also update download.test.ts https://github.com/grafana/grafana/commit/af3ea54d0e8a5c84e25d6473bc7a2f53b6cdd960#diff-f4d98a123cd5f0f5b9f5ce16bb57b66882cffbbcb9ef6624aed059d836d304bd

Reply:
thanks. bro. i will modified it.